metagov/metagov/plugins/tsc/models.py

```diff
@@ -0,0 +1,87 @@
+import metagov.core.plugin_decorators as Registry
+from metagov.core.models import Plugin
+
+import requests
+
+@Registry.plugin
+class TSC(Plugin):
```

Comment:
This looks good to me! TSC stands for Twitter Social Capital, right? There's nothing Twitter-specific about this code, it seems like it could be used to get any kind of contract data from the agreement engine. So it might make more sense to call this plugin `AgreementEngine` instead.

Reply:
This plugin is actually just for TSC! The agreements prototype on GitHub is solely a TSC implementation (although it does have agreements using TSC built into it). Currently, the plan is to move the agreements into the AE Driver and use TSC as an enforcement mechanism, so the API will change in the future. However, this plugin will always be for TSC.